README.md

```diff
@@ -59,38 +45,40 @@ cd spongebob-cli; sudo chmod +x spongebob-cli; sudo python setup.py install
  --help | -h      usage --help this will print what each argument does.
 ```
 
-------
 ## Arch and arch based distributions
 This is not keept up to date regulary.
 There is an [AUR](https://aur.archlinux.org/packages/spongebob-cli-git/) package for spongebob-cli mantained by [getchoo](https://github.com/getchoo)
 ```
 yay -S spongebob-cli-git
 ```
-# Episodes that are known to not work
-Episode: 30, 
 
-------
-# Troubleshooting
+## Episodes that are known to not work
+- Episode 30
 
-- If setup.py fails try this:
-  - ```bash
-    pip3 install termcolor beautifulsoup4 prettytable halo
-    ```
+## Troubleshooting
 
-- If you're getting an error like <code>`sudo: python: command not found`</code>
-   - ```bash
-     git clone https://github.com/trakBan/spongebob-cli.git; cd spongebob-cli; sudo chmod +x spongebob-cli; sudo python3 setup.py install
-     ```
-   - Use python3 instead of python
+### If setup.py fails try this:
```

Comment:
Titling issues like this makes them stand out more, and formatting the solution better also makes it easier to see/copy and paste